package.json

```diff
@@ -76,7 +77,8 @@
     "status:subgraph-sokol-green": "yarn --cwd packages/cardpay-subgraph sokol-green-status",
     "status:subgraph-sokol-blue": "yarn --cwd packages/cardpay-subgraph sokol-blue-status",
     "status:subgraph-xdai-green": "yarn --cwd packages/cardpay-subgraph xdai-green-status",
-    "status:subgraph-xdai-blue": "yarn --cwd packages/cardpay-subgraph xdai-blue-status"
+    "status:subgraph-xdai-blue": "yarn --cwd packages/cardpay-subgraph xdai-blue-status",
+    "w-var": "node ./scripts/w-var.js"
```

Comment:
I'm also ok with naming this command `waypoint-vars` to make it clearer when glancing over package.json commands

Reply:
That seems good, I wasn't sure if that would be too verbose but I think it makes sense to make it clear rather than save on typing a few letters.